srsenb/src/stack/rrc/rrc_ue.cc

```diff
@@ -1046,12 +1046,24 @@ bool rrc::ue::setup_erabs(const asn1::s1ap::erab_to_be_setup_list_ctxt_su_req_l&
 
 bool rrc::ue::release_erabs()
 {
+  for (const auto& erab_it : bearer_list.get_erabs()) {
+    bearer_list.rem_gtpu_bearer(erab_it.second.id);
```

Comment:
the `rem_gtpu_bearer` is now in `release_21_04_final` automatically called in `bearer_list.release_erabs()`, so this line can be removed.

Reply:
Done